# Outputs distance in m
Header header
float64 front
float64 rear
float64 left_front
float64 left_rear
float64 right_front
float64 right_rear
